float64 cam_lift_goal
---
bool cam_lift_result
---
bool cam_lift_feedback

